tool/llpcAutoLayout.cpp

```diff
@@ -461,10 +468,25 @@ void DoAutoLayoutDesc(
         default:
             {
                 SPIRVWord binding = SPIRVID_INVALID;
-                SPIRVWord descSet = SPIRVID_INVALID;
-                if (pVar->hasDecorate(DecorationBinding, 0, &binding) &&
-                    pVar->hasDecorate(DecorationDescriptorSet, 0, &descSet))
+                SPIRVWord descSet = 0;
+                if (pVar->hasDecorate(DecorationBinding, 0, &binding))
                 {
+                    // This copes with a variable having a binding but no set. I saw that happen in
+                    // shaderdb/OpDecorationGroup_TestGroupAndGroupMember_lit.spvas
+                    pVar->hasDecorate(DecorationDescriptorSet, 0, &descSet);
+
+                    // Find/create the node entry for this set and binding.
+                    ResNodeSet& set = resNodeSets[descSet];
+                    auto iteratorAndCreated = set.bindingMap.insert({binding, set.nodes.size()});
```

Comment:
iteratorAndCreated  -> it

Reply:
Disagree -- this is not an iterator, it is a std::pair of an iterator and a flag saying whether insert() created a new entry in the map. So my name makes sense.